homework-submission/myapp/src/App.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
+import './App.css';
+import Category from './Category';
+
+class App extends Component {
+  render() {
+    return (
+      <div className="App">
+        <header className="App-header">
+          
+          <p>
+            Todo<span className='designLogo'>App</span>
+          </p>
+          
+        </header>
+
+        <Category />
```

Comment:
This functionality works, but you didn't accomplish what is needed by the [homework](https://github.com/HackYourFuture/React/blob/master/documentation/homework/W1-homework.md).

Homework says: **You have to use more than two components. Think which props the components should take.**

Which means create "<Item deadline="" description="" />" component and pass the props to it.